src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = {};
+
+  sourceString.replace(/\n/g, '')
+    .split(';')
+    .map(property => property.trim())
+    .filter(property => property !== '')
+    .map(property => property.split(':'))
+    .map(property => {
+      properties[property[0].trim()] = property[1].trim();
+
+      return property;
```

Comment:
You don't need to return `property` here
```suggestion
      properties[property[0].trim()] = property[1].trim();
```

Reply:
yeah, fixed
I'm used to the linter quarreling without a return on the Mate platform)